### ISSUE TITLE
[Accessibility][Product Carousel] The "next" and "previous" buttons for the carousel do not have accessible names

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1765]
**Steps to Reproduce**
1. With NVDA running, open page in Google Chrome.
2. Navigate to the previous or next control for the product carousel.

**Current Behavior**
The screen reader announces the controls simply as "button," given that the buttons have no text content.

**Expected behavior/code**
The operable buttons to move to the previous selection or next selection in the carousel should have accessible names. 

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Assistive technology users

**WCAG SC**
1.3.1 Info and Relationships

**Possible Solution**
Add accessible names with visually-hidden content within the buttons, or with aria-label attributes on the buttons.

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES

**Comment by mhaack** [comment]
Fixed by CIF-2279 via https://github.com/adobe/aem-core-cif-components/pull/666
